pkg/acs/calacs/lib/acshist.c

```diff
@@ -169,28 +169,28 @@ int blevHistory (ACSInfo *acs, Hdr *phdr, int done, int driftcorr) {
 	return (status);
 }
 
-int sinkHistory (ACSInfo *acs, Hdr *phdr) {
+int sinkHistory (const ACSInfo *acs, Hdr *phdr) {
 
     extern int status;
 
-    int logit;			/* true if we should log file name */
+    int logit; // True if we should log file name
     int OmitStep (int);
-    int ImgHistory (RefImage *, Hdr *);
+    int ImgHistory (const RefImage *, Hdr *);
     int UpdateSwitch (char *, int, Hdr *, int *);
 
     if (OmitStep (acs->sinkcorr))
-        return (status);
+        return (status = HSTCAL_OK);
```

Comment:
I still don't understand why you force `HSTCAL_OK` here. This line is only executed when something goes wrong in `OmitStep`.